hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -375,6 +377,31 @@ class TestHoodieSparkSqlWriter {
     assert(df.except(trimmedDf).count() == 0)
   }
 
+  /**
+   * Test case for insert dataset without partitioning field
+   */
+  @Test
+  def testInsertDatasetWithoutPartitionField(): Unit = {
```

Comment:
Can we add a similar test for non-partitioned table and validate table properties may be.

Reply:
Sorry, didn't get your comment: what kind of test were you referring to?